playbooks/openedx_native.yml

```diff
@@ -67,6 +67,15 @@
     # For the mfe role.
     COMMON_ECOMMERCE_BASE_URL: '{{ ECOMMERCE_ECOMMERCE_URL_ROOT }}'
     ECOMMERCE_ENABLE_PAYMENT_MFE: true
+    ENABLE_UPTIME_REPORT: true
+    RETIREMENT_SERVICE_GIT_REPOS:
+      - PROTOCOL: "{{ COMMON_GIT_PROTOCOL }}"
+        DOMAIN: "{{ COMMON_GIT_MIRROR }}"
+        PATH: "open-craft"
+        REPO: "tubular.git"
+        VERSION: "opencraft-release/koa.3"
```

Comment:
@pomegranited what are your thoughts here?  We can't use open-release/lilac.2 for tubular because https://github.com/edx/tubular/pull/531 didn't make it to lilac.2, but we should default to a lilac release here to avoid issues.  Maybe we can default to open-release/lilac.master here?

Reply:
:+1:  I updated in https://github.com/open-craft/configuration/pull/175/commits/06fb8f4f4921601695586f080c2a6e68045c1602 - would you mind sanity checking this?

I'm also deploying a [new sandbox instance](https://manage.opencraft.com/instance/29580/edx-appserver/21078/) for testing.  CC @michaelbukachi if you'd like to make another review pass now that there has been some changes.